doc/content/getstarted.md

```diff
@@ -2,62 +2,82 @@
 title: Get Started
 ---
 
-## The Scientific Python theme for Hugo
-
 The **Scientific Python Hugo Theme** is a theme for the
 [Hugo](https://gohugo.io) static site generator based on the
```

Comment:
I am not sure I would say based on pydata-sphinx-theme. Maybe something like with a similar look and feel, or uses some of the CSS from pydata-sphinx-theme.